server/src/main/java/game/shooting/GameShootingPart.java

```diff
@@ -0,0 +1,53 @@
+package game.shooting;
+
+import game.shooting.matchers.PlayerBoardMatcher;
+import game.shooting.matchers.PlayerToShotSupplierMatcher;
+import model.preparing.Player;
+import model.shooting.board.ShootingBoard;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GameShootingPart implements ObservableGame {
```

Comment:
All public classes/methods should have javadocs

Reply:
This issue is tracked in number #38 and will be resolved later on.